config/application-configuration.json

```diff
@@ -23,6 +23,17 @@
         "optional": true,
         "description": "The temperature of the cool fluid returning to the heat pump in degrees Celsius (°C)"
     },
+    "RoomTemperatureZone1": {
+        "displayOptions": {
+            "label": "Inside",
+            "color": "purple"
```

Comment:
Need to add `"fixed" : 1` here so that temperature is rounded to 1 decimal place.

Reply:
Oh, and the same in "TargetHCTemperatureZone1" too.